flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/src/main/java/org/apache/flink/cdc/connectors/doris/sink/DorisSchemaChangeManager.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.cdc.connectors.doris.sink;
+
+import org.apache.doris.flink.catalog.doris.FieldSchema;
+import org.apache.doris.flink.cfg.DorisOptions;
+import org.apache.doris.flink.exception.IllegalArgumentException;
+import org.apache.doris.flink.sink.schema.SchemaChangeManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Provides schema change operations based on {@link SchemaChangeManager}. */
+public class DorisSchemaChangeManager extends SchemaChangeManager {
+    private static final Logger LOG = LoggerFactory.getLogger(DorisSchemaChangeManager.class);
+
+    public DorisSchemaChangeManager(DorisOptions dorisOptions) {
+        super(dorisOptions);
+    }
+
+    private static final String CREATE_DATABASE_DDL = "CREATE DATABASE %s %s %s;";
+    private static final String MODIFY_COLUMN_DDL = "ALTER TABLE %s MODIFY COLUMN %s %s";
+
+    public boolean createDatabase(String database) throws IOException, IllegalArgumentException {
+        return createDatabase(database, new HashMap<>(), true);
+    }
+
+    public boolean createDatabase(String database, Map<String, String> properties)
+            throws IOException, IllegalArgumentException {
+        return createDatabase(database, properties, true);
+    }
+
+    public boolean createDatabase(
+            String database, Map<String, String> properties, boolean ignoreIfExists)
+            throws IOException, IllegalArgumentException {
+        String createDatabaseDDL =
+                String.format(
+                        CREATE_DATABASE_DDL,
+                        ignoreIfExists ? "IF NOT EXISTS" : "",
+                        database,
```

Comment:
org.apache.doris.flink.catalog.doris.DorisSystem has already implemented `createDatabase` , it can be called directly.